Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -922,10 +932,17 @@ class SettingsTableViewController: UITableViewController {
 
     override func tableView(tableView: UITableView, heightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
         //make account/sign-in and close private tabs rows taller, as per design specs
-        if ((indexPath.section == 0 && indexPath.row == 0) ||
-            (indexPath.section == 2 && indexPath.row == 1)) {
-            return 64
+        if #available(iOS 9, *) {
+            if ((indexPath.section == 0 && indexPath.row == 0) ||
+                (indexPath.section == 2 && indexPath.row == 1)) {
+                return 64
+            }
+        } else {
+            if indexPath.section == 0 && indexPath.row == 0 {
+                return 64
+            }
```

Comment:
Maybe reduce this a bit:
```
if indexPath.section == 0 && indexPath.row == 0 {
    return 64
}

if #available(iOS 9, *) {
    if indexPath.section == 2 && indexPath.row == 2 {
        return 64
    }
}
```

On Android, we simplified these kinds of things even more by extracting preprocessed bits into a single constants file, so then we could just do `AppConstants.IsIOS9`, or something similar, and it could be used as a regular `Bool`. Not saying we need to do that here, but just something we might want to think about!